src/DistributedMetadata/DDLService.cpp

```diff
@@ -260,6 +266,31 @@ Int32 DDLService::doDDL(
     return err;
 }
 
+void DDLService::doDDLOnHosts(const std::vector<Poco::URI> & target_hosts, const String & payload,
+                              const String & method, const String & query_id, const String & user) const
+{
+    std::vector<String> failed_hosts;
+    /// FIXME : Parallelize doDDL on the uris
+    for (auto uri : target_hosts)
```

Comment:
`for (const auto & uri : target_hosts)` ?

Reply:
fixed